data/data/install.openshift.io_installconfigs.yaml

```diff
@@ -3009,6 +3009,10 @@ spec:
                           type: string
                         type: array
                     type: object
+                  internalLBIP:
+                    description: InternalLBIP specifies the IP address of the internal
+                      LB to be set.
```

Comment:
Nit: InternalLBIP specifies the IP address to be used for the Internal Load Balancer.